static/app/types/group.tsx

```diff
@@ -352,6 +352,10 @@ export interface GroupActivityAssigned extends GroupActivityBase {
     assigneeType: string;
     user: Team | User;
     assigneeEmail?: string;
+    /** If the user was assigned via an integration */
```

Comment:
nit, this comment style
```suggestion
    /**
     * If the user was assigned via an integration
     */
```